packages/slice-machine/components/Changelog/index.tsx

```diff
@@ -15,11 +15,14 @@ export default function Changelog() {
     })
   );
 
-  // Null is when no version are found (edge case)
   const [selectedVersion, setSelectedVersion] = useState<PackageVersion | null>(
     changelog.versions[0] || null
   );
 
+  useEffect(() => {
+    setSelectedVersion(changelog.versions[0]);
+  }, [JSON.stringify(changelog)]);
```

Comment:
why the json stringify ?

Reply:
We can't just use changelog because the shallow comparison doesn't see them as equal. So causes a infinite renders. But it can compare 2 strings 😕